src/plugins/pluginMFManifest.ts

```diff
@@ -1,42 +1,78 @@
-import { join, relative } from 'pathe';
-import { Manifest, Plugin } from 'vite';
+import * as path from 'pathe';
+import { Plugin } from 'vite';
+import type { PluginContext } from 'rollup';
 import {
   getNormalizeModuleFederationOptions,
   getNormalizeShareItem,
 } from '../utils/normalizeModuleFederationOptions';
-import { getPreBuildLibImportId, getUsedRemotesMap, getUsedShares } from '../virtualModules';
+import { getUsedRemotesMap, getUsedShares } from '../virtualModules';
+
+import {
+  buildFileToShareKeyMap,
+  collectCssAssets,
+  createEmptyAssetMap,
+  deduplicateAssets,
+  JS_EXTENSIONS,
+  PreloadMap,
+  processModuleAssets,
+  trackAsset,
+} from '../utils/cssModuleHelpers';
+
+// Helper to build share key map with proper context typing
+interface BuildFileToShareKeyMapContext {
+  resolve: PluginContext['resolve'];
+}
 
 const Manifest = (): Plugin[] => {
   const mfOptions = getNormalizeModuleFederationOptions();
   const { name, filename, getPublicPath, manifest: manifestOptions } = mfOptions;
+
   let mfManifestName: string = '';
   if (manifestOptions === true) {
     mfManifestName = 'mf-manifest.json';
   }
   if (typeof manifestOptions !== 'boolean') {
-    mfManifestName = join(manifestOptions?.filePath || '', manifestOptions?.fileName || '');
+    mfManifestName = path.join(manifestOptions?.filePath || '', manifestOptions?.fileName || '');
   }
-  let extensions: string[];
+
   let root: string;
-  type PreloadMap = Record<
-    string,
-    {
-      sync: string[];
-      async: string[];
-    }
-  >; // 保存模块和文件的映射关系
   let remoteEntryFile: string;
   let publicPath: string;
   let _command: string;
   let _originalConfigBase: string | undefined;
   let viteConfig: any;
+
+  /**
+   * Adds global CSS assets to all module exports
+   * @param filesMap - The preload map to update
+   * @param cssAssets - Set of CSS asset filenames to add
+   */
+  const addCssAssetsToAllExports = (filesMap: PreloadMap, cssAssets: Set<string>) => {
+    Object.keys(filesMap).forEach((key) => {
+      cssAssets.forEach((cssAsset) => {
+        trackAsset(filesMap, key, cssAsset, false, 'css');
+      });
+    });
+  };
```

Comment:
@pcfreak30 You forgot to update this codepath to use `addCssAssetsToAllExports` from `cssModuleHelpers`.

Reply:
@pcfreak30 you have `addCssAssetsToAllExports` implemented in `cssModuleHelpers.ts`, but yet you have an exact copy of it here, in `pluginMFManifest.ts`